parser-typechecker/src/Unison/Runtime/Builtin.hs

```diff
@@ -1844,48 +1869,48 @@ declareForeigns = do
       asCert :: PEM -> Either String X.SignedCertificate
       asCert pem = X.decodeSignedCertificate  $ pemContent pem
     in
-      declareForeign "Tls.decodeCert.impl.v3" boxToEFBox . mkForeignTls $
+      declareForeign Tracked "Tls.decodeCert.impl.v3" boxToEFBox . mkForeignTls $
         \(bytes :: Bytes.Bytes) -> pure $ mapLeft wrapFailure $ (decoded >=> asCert) bytes
 
-  declareForeign "Tls.encodeCert" boxDirect . mkForeign $
+  declareForeign Tracked "Tls.encodeCert" boxDirect . mkForeign $
     \(cert :: X.SignedCertificate) -> pure $ Bytes.fromArray $ X.encodeSignedObject cert
 
-  declareForeign "Tls.decodePrivateKey" boxDirect . mkForeign $
+  declareForeign Tracked "Tls.decodePrivateKey" boxDirect . mkForeign $
     \(bytes :: Bytes.Bytes) -> pure $ X.readKeyFileFromMemory $ L.toStrict $ Bytes.toLazyByteString bytes
 
-  declareForeign "Tls.encodePrivateKey" boxDirect . mkForeign $
+  declareForeign Tracked "Tls.encodePrivateKey" boxDirect . mkForeign $
     \(privateKey :: X.PrivKey) -> pure $ Util.Text.pack $ show privateKey
 
-  declareForeign "Tls.receive.impl.v3" boxToEFBox . mkForeignTls $
+  declareForeign Tracked "Tls.receive.impl.v3" boxToEFBox . mkForeignTls $
     \(tls :: TLS.Context) -> do
       bs <- TLS.recvData tls
       pure $ Bytes.fromArray bs
 
-  declareForeign "Tls.terminate.impl.v3" boxToEFBox . mkForeignTls $
+  declareForeign Tracked "Tls.terminate.impl.v3" boxToEFBox . mkForeignTls $
     \(tls :: TLS.Context) -> TLS.bye tls
 
-  declareForeign "Code.dependencies" boxDirect
+  declareForeign Untracked "Code.dependencies" boxDirect
     . mkForeign $ \(sg :: SuperGroup Symbol)
         -> pure $ Wrap Ty.termLinkRef . Ref <$> groupTermLinks sg
-  declareForeign "Code.serialize" boxDirect
+  declareForeign Untracked "Code.serialize" boxDirect
     . mkForeign $ \(sg :: SuperGroup Symbol)
         -> pure . Bytes.fromArray $ serializeGroup sg
-  declareForeign "Code.deserialize" boxToEBoxBox
+  declareForeign Untracked "Code.deserialize" boxToEBoxBox
     . mkForeign $ pure . deserializeGroup @Symbol . Bytes.toArray
-  declareForeign "Code.display" boxBoxDirect . mkForeign
+  declareForeign Untracked "Code.display" boxBoxDirect . mkForeign
     $ \(nm,sg) -> pure $ prettyGroup @Symbol (Util.Text.unpack nm) sg ""
```

Comment:
Does this function expose any details about names for hashes that may exist in the local codebase? Or is it just showing hashes and local variable names? If the latter then I think it's fine to be `Untracked`, otherwise `Tracked`.

Reply:
It appears to just show hashes and local variables right now. I don't know if there are plans to change that in the future, though.

Reply:
K, then I think this is fine for now.